### ISSUE TITLE
DNS_PROBE_POSSIBLE On UI Page

### ISSUE THREAD

**Issue by ZaineJJ** [dappnode/DAppNodePackage-Hopr#56]
Version: 1.0.0 (1.90.12 upstream) [/ipfs/QmaGNSnRHduxb98FzZULnN9cuQTChybAUXdTw6Pza1b958]

When click on UI on DAppNode 
directed to the page
http://hopr.dappnode:3000/?apiEndpoint=http://hopr.dappnode:3001&apiToken=!5qxc9Lp1BE7IFQ-nrtttU

But 
![image](https://user-images.githubusercontent.com/88610565/193377177-35d95f4a-4c4b-468d-87c5-9b728181513e.png)

Unable to go into the UI and logs shows

> 2022-10-01T00:43:19.397Z hoprd:admin using /app/packages/hoprd/hopr-admin
> 2022-10-01T00:43:19.749Z hoprd { type: 'log', msg: 'Admin server listening on port 3000', ts: '2022-10-01T00:43:19.747Z' }
> 2022-10-01T00:43:19.750Z hoprd { type: 'log', msg: 'This is HOPRd version 1.90.12', ts: '2022-10-01T00:43:19.750Z' }
> 2022-10-01T00:43:19.750Z hoprd { type: 'log', msg: 'This node appears to be running on an AVADO/Dappnode', ts: '2022-10-01T00:43:19.750Z' }
> 2022-10-01T00:43:21.844Z hoprd { type: 'log', msg: 'Creating HOPR Node', ts: '2022-10-01T00:43:21.844Z' }
> 2022-10-01T00:43:21.846Z hopr-core:db using db at /app/hoprd-db/data/db
> 2022-10-01T00:43:21.846Z hopr-core:db db directory does not exist, creating?: true
> 2022-10-01T00:43:21.856Z hopr-core:db namespacing db by public key: 0x02ef7c0178c866a13aa45e0978634d998db8e79897191f54b28a005d52501d5a2b
> 2022-10-01T00:43:21.857Z hopr-core:db setting environment id monte_rosa to db
> 2022-10-01T00:43:21.857Z hopr-core:create-hopr using provider URL: https://provider-proxy.hoprnet.workers.dev/xdai_mainnet
> 2022-10-01T00:43:21.858Z hopr-core using environment: monte_rosa
> 2022-10-01T00:43:21.859Z hoprd { type: 'status', msg: 'PENDING', ts: '2022-10-01T00:43:21.859Z' }
> 2022-10-01T00:43:21.859Z hoprd { type: 'log', msg: 'Ready to request on-chain connector to connect to provider.', ts: '2022-10-01T00:43:21.859Z' }
> 2022-10-01T00:43:22.523Z hoprd { type: 'log', msg: 'Connector has been loaded properly.', ts: '2022-10-01T00:43:22.523Z' }
> 2022-10-01T00:43:22.523Z hoprd:api Enabling Rest API v2 and WS API v2
> 2022-10-01T00:43:22.763Z hoprd { type: 'log', msg: 'Node address: 16Uiu2HAmBYX6EeLVKjDJ5fpBXpFhGHaXeHGE4QtFFh4YAALLkUuG', ts: '2022-10-01T00:43:22.763Z' }
> 2022-10-01T00:43:22.763Z hoprd { type: 'log', msg: 'Node is not started, please fund this node 0x591EcBC5Db6B20B8a93013a2c29CAd044cF6390c with at least 0.01 xDAI', ts: '2022-10-01T00:43:22.763Z' }
> 2022-10-01T00:43:22.767Z hoprd { type: 'log', msg: 'API server on 0.0.0.0 listening on port 3001', ts: '2022-10-01T00:43:22.766Z' }
> 2022-10-01T00:43:22.767Z hoprd { type: 'log', msg: 'Healthcheck server on 0.0.0.0 listening on port 8080', ts: '2022-10-01T00:43:22.767Z' }
> 2022-10-01T00:43:23.317Z hoprd:api:v2 Written full Rest API v2 spec file to rest-api-v2-full-spec.json
> 2022-10-01T00:43:24.140Z hopr-core still unfunded, trying again soon
> 2022-10-01T00:43:24.140Z hopr:utils:retry failed, attempting again in 1000 (undefined)
> 2022-10-01T00:43:26.405Z hopr-core still unfunded, trying again soon
> 2022-10-01T00:43:26.405Z hopr:utils:retry failed, attempting again in 1050 (undefined)
> 2022-10-01T00:43:28.326Z hopr-core still unfunded, trying again soon
> 2022-10-01T00:43:28.326Z hopr:utils:retry failed, attempting again in 1102.5 (undefined)

**Comment by ZaineJJ** [comment]
> For some reason the dns is not resolved it by default, I fixed it by changing the URL to `http://node.hopr.dappnode:3000/?apiEndpoint=http://node.hopr.dappnode:3001&apiToken=!5qxc9Lp1BE7IFQ-nrtttU`
> 
> By default if the package has one service or one of the service is defined as the main service in the manifest, the URL to access would be namepackage.dappnode and service-name.namepackage.dappnode if you want to access to another service via URL.

I am still facing DNS probe issue 
![image](https://user-images.githubusercontent.com/88610565/194296872-9b0c8bda-e73f-493e-bce5-1f2cab307c58.png)

**Comment by tropicar** [comment]
https://github.com/dappnode/DAppNodePackage-Hopr/pull/61
With this change it should work, the URL to access are :
http://node.hopr.public.dappnode:8080/healthcheck/v1/version
http://node.hopr.public.dappnode:3000/?apiEndpoint=http://node.hopr.public.dappnode:3001&apiToken=!5qxc9Lp1BE7IFQ-nrtttU

**Comment by ZaineJJ** [comment]
Yes I clicked on it and it's working now, awesome! Much appreciated @tropicar for the fix :D
